plugins/kubernetes/README.md

```diff
@@ -308,3 +308,11 @@ metadata.annotation.samson/force_update: "true"
 ### Static config per deploy group
 
 Set the kubernetes roles to `kubernetes/$deploy_group/server.yml` 
+
+### Ignoring warning events
+
+If a warning event stops deploys, but they are safe to ignore, add this:
+
+`metadata.annotations.samson/ignore_events="FailedCreate"`
```

Comment:
```suggestion
`metadata.annotations.samson/ignore_events="FailedCreate,AnotherEvent"` 
```

To show there might be multiple and show separator.

Reply:
good idea!